### PR TITLE
Add WeChat Online

### LINKED ISSUE #176: Add WeChat Online support
Add at the `CallMethod` enum the `web` and `app` values

Add at the `OnlineBrand` enum the WeChat 

Add to the transaction token creation the `user_identifier`, it is optional String field.

### PR DESCRIPTION
**Pull request by yukimogu**
Closes #176 
Add unimplemented other brands
Fix to be able to add body with POST

### CODE REVIEW

File: src/main/java/com/univapay/sdk/types/brand/OnlineBrand.java [hunk 1]
```diff
@@ -3,10 +3,30 @@
...
+  @SerializedName("easypaisa")
+  EASYPAISA,
+  @SerializedName("gcash")
+  GCASH,
+  @SerializedName("kakaopay")
+  KAKAOPAY,
+  @SerializedName("tng")
+  TOUCH_N_GO,
+  @SerializedName("truemoney")
+  TRUEMONEY,
```
Comment (andrezimmermann) [inline src/main/java/com/univapay/sdk/types/brand/OnlineBrand.java:31]: These are QR Code brands so I think they don't need to be here 👍

**Review by andrezimmermann — CHANGES_REQUESTED** [review]
LGTM, just need to adjust those QR Brands! 😃

File: src/main/java/com/univapay/sdk/models/common/CallMethod.java [hunk 2]
```diff
@@ -13,5 +13,9 @@
   HTTP_POST,
   /** The issuer token should be forwarded to the service's provided SDK */
   @SerializedName("sdk")
-  SDK
+  SDK,
+  @SerializedName("WEB")
+  WEB,
+  @SerializedName("APP")
```
Comment (kgston) [inline src/main/java/com/univapay/sdk/models/common/CallMethod.java:19]: These should be lowercase

**Review by kgston — COMMENTED** [review]
Minor stuff 👍

File: src/main/java/com/univapay/sdk/models/common/CallMethod.java (hunk 2 above)
Comment (andrezimmermann) [reply src/main/java/com/univapay/sdk/models/common/CallMethod.java:19]: Right, thanks for catching that 👍

**Review by andrezimmermann — COMMENTED** [review]
Thanks, will merge and release this soon 😄

**Review by andrezimmermann — APPROVED** [review]

**Pull request merged**